sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py

```diff
@@ -114,18 +114,18 @@ def test_evaluate_with_groundedness_evaluator(self, model_config, data_file):
         assert result["rows"] is not None
         assert row_result_df.shape[0] == len(input_data)
 
-        assert "outputs.grounded.gpt_groundedness" in row_result_df.columns.to_list()
+        assert "outputs.grounded.groundedness" in row_result_df.columns.to_list()
         assert "outputs.f1_score.f1_score" in row_result_df.columns.to_list()
 
-        assert "grounded.gpt_groundedness" in metrics.keys()
+        assert "grounded.groundedness" in metrics.keys()
```

Comment:
Do we also need a change here.
https://msdata.visualstudio.com/Vienna/_git/workspace-portal?path=%2Fcommon%2Fevaluation%2Fsrc%2FConstants.ts&version=GCb75b6ced89c2872bc2a12ea4479a50dde9ecf52c&_a=contents

Reply:
No - we're not removing anything from the outputs (yet). We're adding a new value (duplicate value, different key name). UI is unaffected as the old key is still there for them to index.

Eventually, we will deprecate and remove the old key and then UI will need to change the key.